### PR TITLE
tune pdd rules #52

### PR DESCRIPTION
**Pull request by artemlazarev**
tune pdd rules #52

### CODE REVIEW

**Comment by 0crat** [comment]
@artemlazarev[/z](https://www.0crat.com/u/artemlazarev) this pull request is too small, just 4 lines changed (less than 10), there will be no formal code review, see [§53](http://www.zerocracy.com/policy.html#53) and [§28](http://www.zerocracy.com/policy.html#28); in the future, try to make sure your pull requests are not too small; @g4s8[/z](https://www.0crat.com/u/g4s8) please review this and merge or reject

File: .pdd [hunk 1]
```diff
@@ -5,6 +5,6 @@
 --exclude src/test/resources-binary/**/*
 --exclude src/test/resources/example_pkg/dist/**/*
 --exclude .idea/**/*
---rule min-words:15
+--rule min-words:7
```
Comment (g4s8) [inline .pdd:8]: @artemlazarev I don't think we need very short descriptions for tasks, it would be hard to understand what to do next

Comment (artemlazarev) [reply .pdd:8]: @g4s8 
The example that completely describe that we must to do in 8 words
" * @todo #33:90min Increase version of "com.artipie:http" at least to 0.11"

Comment (g4s8) [reply .pdd:8]: @artemlazarev I've already updated `artipie/http` dependency to the latest version, I don't think we need such small tasks to be submitted as a separate tickets.

Comment (artemlazarev) [reply .pdd:8]: @g4s8 this is example of complete and clean description in pdd in 8 word. And that not about concrete puzzle, thats an example.

**Comment by 0crat** [comment]
Job `gh:artipie/pypi-adapter#53` is not assigned, can't get performer

**Pull request closed without merging**